luarules/gadgets/cmd_undo.lua

```diff
@@ -9,10 +9,14 @@ function gadget:GetInfo()
 		date	= 'June 2017',
 		license	= 'GNU GPL, v2 or later',
 		layer	= 1,
-		enabled	= Spring.GetTeamList()-1 > 64
+		enabled	= true
 	}
 end
 
+if Spring.GetTeamList()-1 <= 64 then
```

Comment:
I think this will still crash
You're doing `table - 1 <= 64`
You gotta be doing `#table - 1 <= 64`